acestep/audio_utils.py

```
@@ -1,354 +1,406 @@
+            output_path: Output audio file path
+            output_format: Target format ('flac', 'wav', 'mp3')
+            remove_input: Whether to delete input file
+        
+        Returns:
+            Output file path
+        """
+        input_path = Path(input_path)
+        output_path = Path(output_path)
+        
+        if not input_path.exists():
+            raise FileNotFoundError(f"Input file not found: {input_path}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)